esp/templates/program/modules/surveymanagement/manage.html

```diff
@@ -0,0 +1,176 @@
+        <table align="center" cellpadding="0" cellspacing="0" width="100%">
+            <tr><th colspan="2">Add/modify a Survey</th></tr>
+            {{ survey_form.as_table }}
+            <tr><td colspan="2" align="center"><input class="fancybutton" type="submit" value="Create/Save Survey" /></td></tr>
+        </table>
+    </form>
+    <br />
+    <table align="center" cellpadding="0" cellspacing="0" width="100%">
+        <tr><th colspan="3">Surveys for {{ program.niceName }}</th></tr>
+        <tr><td colspan="3"><table width="100%" cellpadding="0" cellspacing="0">
+        <tr>
+            <td width="75%" class="underline"><b>Name</b></td>
+            <td width="15%" class="underline"><b>Category</b></td>
+            <td width="10%" class="underline"></td>
+        </tr>
+        {% for survey in surveys %}
+            <tr>
+                <td class="underline">{{ survey.name }}</td>
+                <td class="underline">{{ survey.category }}</td>
+                <td class="underline">
+                    <a href="/manage/{{ program.url }}/surveys/manage?obj=survey&op=edit&id={{ survey.id }}">[Edit]</a>
+                    <a href="/manage/{{ program.url }}/surveys/manage?obj=survey&op=delete&id={{ survey.id }}">[Delete]</a>
+                </td>
+            </tr>
+        {% endfor %}
+        </table></td></tr>
+    </table>
+    </div>
+    {% ifequal open_section "question" %}
+    <button class="dsphead active">
+       <b>Step 2: Add Questions</b> (click to expand/contract)
+    </button>
+
+    <div class="dspcont active">
+    {% else %}
+    <button class="dsphead">
+       <b>Step 2: Add Questions</b> (click to expand/contract)
+    </button>
+
+    <div class="dspcont">
+    {% endifequal %}
+    <form method="post" action="/manage/{{ program.url }}/surveys/manage?obj=question">
+        {% if question_form.id %}
+        <input type="hidden" name="question" value="{{ question_form.id }}" />
+        {% endif %}
+        <table align="center" cellpadding="0" cellspacing="0" width="100%">
+            <tr><th colspan="2">Add/modify a Question</th></tr>
+            {{ question_form.as_table }}
+            <tr><td colspan="2" align="center"><input class="fancybutton" type="submit" value="Create/Save Question" /></td></tr>
+        </table>
+    </form>
+    <br />
+    <table align="center" cellpadding="0" cellspacing="0" width="100%">
+        <tr><th colspan="6">Questions for {{ program.niceName }}</th></tr>
+        <tr>
+            <th class="selector active" colspan="3" width="50%" data-panel="details">Question Details</th>
+            <th class="selector" colspan="3" width="50%" data-panel="rendered">Rendered Questions</th>
+        </tr>
+        <tr id="details-panel"><td colspan="6"><table width="100%" cellpadding="0" cellspacing="0">
+        <tr>
+            <td width="5%"><b>Seq</b></td>
+            <td width="30%"><b>Name</b></td>
+            <td width="25%"><b>Question Type</b></td>
```

Comment:
For the question types that don't have anything after "includes," can we remove the word "includes"?

![image](https://user-images.githubusercontent.com/25392075/77254971-a330aa80-6c3a-11ea-896f-4adbd760d024.png)